### PR TITLE
Add six minutes

### PR DESCRIPTION
**Pull request by falbru**
Not 100% happy with the design. Looks kind of goofy. 100% open for suggestions.

image

image

image

image

### CODE REVIEW

**Comment by falbru** [comment]
føkk i forgot the actual timer for six minutes lmao

**Comment by falbru** [comment]
done but still kinda ugly

**Comment by jonasdeluna** [comment]
image

I was playing around with having the image be blurred behind the play button (so its impossible to tell what song it is, but add some color to the audio player), but I'm logging off for now.

**Comment by falbru** [comment]
Very nice work, @jonasdeluna :raised_hands:

File: src/app/six-minutes/GameOverModal.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <div className="absolute w-screen z-10 h-screen backdrop-blur-sm animate-fade">
+      <Card className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2">
+        <CardHeader>
+          <h2>🚨 Tiden har gått ut 🚨</h2>
+        </CardHeader>
+        <CardContent>Den gruppen som har tur nå må chugge 🍻</CardContent>
+      </Card>
+    </div>
+  );
+};
```
Comment (Bestem0r) [inline src/app/six-minutes/GameOverModal.tsx:14]: Can we use the `shadcn/ui` component [Alert Dialog](https://ui.shadcn.com/docs/components/alert-dialog) for this instead?

Comment (falbru) [reply src/app/six-minutes/GameOverModal.tsx:14]: I thought about it, but it looked like the only way to open it was with the AlertDialogTrigger? I want it to open automatically after 6 minutes.

**Pull request merged**